runtime/vm/JFRChunkWriter.cpp

```diff
@@ -24,6 +24,10 @@
 
 #if defined(J9VM_OPT_JFR)
 
+#if defined(LINUX)
```

Comment:
Can you make sure the ordering of the includes follows https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#header-files?